src/Umbraco.Web.UI.Client/src/common/directives/components/umbrangeslider.directive.js

```diff
@@ -299,7 +300,17 @@ For extra details about options and events take a look here: https://refreshless
                 });
             });
         }
-
+      function setUpActivePipsHandling() {
+        const pips = sliderInstance.querySelectorAll('.noUi-value');
+        let activePip = [null, null];
+        sliderInstance.noUiSlider.on('update', function (values,handle) {
+          if(activePip[handle]){
+            activePip[handle].classList.remove("noUi-value-active");
+          }
+          activePip[handle] = pips[parseInt(values[handle])-1];
+          activePip[handle].classList.add("noUi-value-active");
+        });
+      }
```

Comment:
I think it would be safer to query pips inside the update function as in the example  here https://refreshless.com/nouislider/examples/#section-pips

as it is possible to update options dynamically inclusive `pips`:
https://refreshless.com/nouislider/more/#section-update

Reply:
Alternatively I think is it possible to get pips from the instance calling `pips()`:
https://refreshless.com/nouislider/pips/

Reply:
I have updated this so that it queries inside the update function. I have only done this here as in the user.controller.js the list of pips already exists and is used elsewhere so it should be safe enough to use there.